package.json

```diff
@@ -13,34 +13,59 @@
   "unpkg": "dist/ionicons.js",
   "collection": "dist/collection/collection-manifest.json",
   "collection:main": "dist/collection/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/types/index.d.ts",
+      "default": "./dist/index.js",
+      "require": "./dist/index.cjs.js"
+    },
+    "./components": {
+      "types": "./dist/components/index.d.ts",
+      "import": "./dist/components/index.js"
+    },
+    "./icons": {
+      "types": "./icons/index.d.ts",
+      "import": "./icons/index.mjs",
+      "require": "./icons/index.js"
+    }
+  },
   "scripts": {
-    "compile.scripts": "tsc -p scripts/tsconfig.json",
-    "build": "npm run compile.scripts && npm run build.files && npm run build.component && npm run collection.copy",
-    "build.files": "node scripts/build.js",
+    "build": "run-s clean build.* prettier",
+    "build.files": "tsx scripts/build.js",
     "build.component": "stencil build",
-    "collection.copy": "node scripts/collection-copy.js",
-    "start": "npm run compile.scripts && npm run build.files && stencil build --dev --watch --serve",
-    "test": "npm run test.spec",
-    "test.spec": "stencil test --spec"
+    "build.collection": "tsx scripts/collection-copy.js",
+    "clean": "rimraf dist components icons www",
+    "prettier": "npm run prettier.base -- --write",
+    "prettier.base": "prettier --cache \"./({bin,scripts,src,test}/**/*.{ts,tsx,js,jsx})|bin/stencil|.github/(**/)?*.(yml|yaml)|*.js\"",
+    "prettier.dry-run": "npm run prettier.base -- --list-different",
+    "start": "run-s build.files start.stencil",
```

Comment:
I am getting this error on `npm run start`:

```bash
npm start


> ionicons@7.4.0 start
> run-s build.files start.stencil


> ionicons@7.4.0 build.files
> tsx scripts/build.js

[Error: ENOENT: no such file or directory, open '/Users/brandyscarney/Developer/ionicons/dist/ionicons/svg/accessibility.svg'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/brandyscarney/Developer/ionicons/dist/ionicons/svg/accessibility.svg'
}
ERROR: "build.files" exited with 1.
```

I ran `npm i && npm run build` prior to this. 

I followed the steps in the [contributing guide](https://github.com/ionic-team/ionicons/blob/a97e71239e4fa020a9a616abdd631319b3ae815c/CONTRIBUTING.md#environment-setup).

Reply:
Does it still happen after running `npm install` again? This was happening to me yesterday but ran fine after the second attempt.

Reply:
It seems to fail every second time you call it, I will fix that.